### PR TITLE
[VUFIND-1624] Remove end-of-lifed WorldCatIdentities code.

### PR DESCRIPTION
**Pull request by demiankatz**
This PR removes code related to the end-of-lifed WorldCatIdentities API. No replacement has been provided by OCLC, so this simply aliases the WorldCatIdentities recommendation module to the "Deprecated" module (to prevent the removal from breaking existing configurations) and eliminates associated support code and tests.

TODO
- [x] Run full test suite
- [x] Resolve [VUFIND-1624](https://openlibraryfoundation.atlassian.net/browse/VUFIND-1624) when merging
- [x] Add changelog note about removal of classes

### CODE REVIEW

File: config/vufind/searches.ini [hunk 1]
```diff
@@ -261,8 +261,8 @@ WorkKeys = year
 ; EDSResultsDeferred:[GET parameter]:[result limit]
 ;       Same as EDSResults, but loaded via AJAX.
 ; EPFResults:[GET parameter]:[result limit]
-;       Display EPF (EBSCO Publication Finder) search results matching the terms 
-;       found in the specified GET parameter (default = "lookfor"), limited to a 
+;       Display EPF (EBSCO Publication Finder) search results matching the terms
+;       found in the specified GET parameter (default = "lookfor"), limited to a
```
Comment (maccabeelevine) [inline config/vufind/searches.ini:265]: I assume sneaking in these whitespace fixes is intentional.  Sorry. :)

**Review by maccabeelevine — CHANGES_REQUESTED** [review]
These changes all seem right?  I did find a few additional references:

- https://github.com/vufind-org/vufind/blob/dev/config/vufind/Search2.ini#L116
- https://github.com/vufind-org/vufind/blob/dev/config/vufind/searches.ini#L514
- And in various fixtures like this, but I don't know if the 1.1 etc indicate these are historical only: https://github.com/vufind-org/vufind/blob/dev/module/VuFind/tests/fixtures/configs/1.1/searches.ini

File: config/vufind/searches.ini (hunk 1 above)
Comment (demiankatz) [reply config/vufind/searches.ini:265]: Not a conscious decision, but my editor does it automatically and I decided it was harmless to include it here. :-)

**Comment by demiankatz** [comment]
Thanks, @maccabeelevine! I've fixed Search2.ini in commit ee0a57e (I suspect I accidentally searched local/config instead of plain config when I was working on this part...oops). The line you mention in searches.ini is already fixed here, though, so I'm not sure how it you ended up flagging it. It should be fine to leave the fixtures alone. Those are used for testing the configuration file upgrade tool, so we want to keep them in their historical form.

**Review by maccabeelevine — APPROVED** [review]
Thanks.  Not sure how I missed the one in searches.ini the first time.

**Pull request merged**